data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -38,6 +38,18 @@ OPENSHIFT_HYPERKUBE_IMAGE=$(podman run --quiet --rm ${release} image hyperkube)
 
 CLUSTER_BOOTSTRAP_IMAGE=$(podman run --quiet --rm ${release} image cluster-bootstrap)
 
+# Now, as early as possible we replace the pause image and reload crio to use it, to ensure
+# that we're using the pause image from our payload just like the primary cluster.
+# The config should match the one generated by the MCO ideally:
+# https://github.com/openshift/machine-config-operator/blob/e861ccb12f09c7c768d51fdf0a17879fcc9a87d5/templates/master/01-master-container-runtime/_base/files/crio.yaml
+# But for now we're just changing the key bits: image and command.
+# Perhaps down the line we change this to run something like:
+# podman run machine-config-daemon bootstrap ... (passing the release image and the host rootfs)
+sed -i -e 's,pause_image *=.*,pause_image = "'${MACHINE_CONFIG_INFRA_IMAGE}'",' /etc/crio/crio.conf
+sed -i -e 's,pause_command *=.*,pause_command = "/usr/bin/pod",' /etc/crio/crio.conf
```

Comment:
Is the ${MACHINE_CONFIG_INFRA_IMAGE} always going to have /usr/bin/pod as a command? seems odd to me that value is configurable but this is hardcoded

Reply:
> Is the ${MACHINE_CONFIG_INFRA_IMAGE} always going to have /usr/bin/pod as a command?

This image was used previously in OpenShift v3 I think; there was a big chain of PRs/discussion in the MCO for this.  Bottom line this approach is matching what we do in the MCO right now. 
One main PR is https://github.com/openshift/machine-config-operator/pull/471

Reply:
I figured I was missing some context haha

Reply:
We do a PR from wking that will help fix this but I want our openshift CI to be functional in crio repo before we pull those changes in to avoid regressions.